src/co/elastic/NotificationManager.groovy

```diff
@@ -74,18 +85,26 @@ def analyzeFlakey(Map params = [:]) {
     def numberOfCreatedtedIssues = 0
     tests.each { k, v ->
       def issue = v
-      if (!v?.trim()) {
-        def issueDescription = buildTemplate([
+      def issueDescription = buildTemplate([
           "template": 'flaky-github-issue.template',
           "testName": k,
           "jobUrl": boURL,
-          "testData": testsErrors?.find { it.name.equals(k) }
-        ])
+          "PR": env.CHANGE_ID?.trim() ? "#${env.CHANGE_ID}" : '',
+          "commit": env.GIT_BASE_COMMIT?.trim() ?: '',
+          "testData": testsErrors?.find { it.name.equals(k) }])
+      if (v?.trim()) {
+        try {
+          issueWithoutUrl = v.startsWith('https') ? v.replaceAll('.*/', '') : v
+          githubCommentIssue(id: issueWithoutUrl, comment: issueDescription)
+        } catch(err) {
+          log(level: 'WARN', text: "Something bad happened when commenting the issue '${v}'. See: ${err.toString()}")
+        }
```

Comment:
Report back to the existing issue with the build details:
- https://github.com/elastic/beats/issues/22171#issuecomment-717184741

![Uploading image.png…]()